### ISSUE TITLE
import_definitions defined in diaphora_ida.py misuses CBinDiff

### ISSUE THREAD

**Issue by 4B5F5F4B** [joxeankoret/diaphora#133]
in function import_definitions defined in diaphora_ida.py misuses CBinDiff to create object tmp_diff, but CBinDiff has no attribute named import_definitions_only.  so exception raises.
```
def import_definitions():
  tmp_diff = diaphora.CBinDiff(":memory:")
  filename = AskFile(0, "*.sqlite", "Select the file to import structures, unions and enumerations from")
  if filename is not None:
    if askyn_c(1, "HIDECANCEL\nDo you really want to import all structures, unions and enumerations?") == 1:
      tmp_diff.import_definitions_only(filename)

```

![exception](https://user-images.githubusercontent.com/19218802/48662585-e3b86b00-eabe-11e8-8e37-c5e3f6fca257.PNG)

I think CIDABinDiff should be used instead of CBinDiff

```
def import_definitions():
  tmp_diff = CIDABinDiff(":memory:")
  filename = AskFile(0, "*.sqlite", "Select the file to import structures, unions and enumerations from")
  if filename is not None:
    if askyn_c(1, "HIDECANCEL\nDo you really want to import all structures, unions and enumerations?") == 1:
      tmp_diff.import_definitions_only(filename)
```

**Comment by joxeankoret** [comment]
You're right. Working on it!

**Comment by joxeankoret** [comment]
Fixed with 6718a29bda7b1395d79f3eac19b2b1807285a1ce
